examples/plot-graph-using-bokeh.ipynb

```diff
@@ -0,0 +1,200 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "92b7c5ca",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "This is an example from [bokeh tutorial](https://github.com/bokeh/jupyter_bokeh/blob/main/examples/jupyter_interactors.ipynb). In this example, we are going to use [Bokeh](https://docs.bokeh.org/en/latest/docs/user_guide/jupyter.html) to plot an interactive graph. We will plot different trignometric functions, namely sin, cos and tan. To interactive with the graph, we will use a dropdown that offers a choice to select a function and the sliders to control the frequency, amplitude, and phase."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e55082b",
+   "metadata": {},
+   "source": [
+    "**Install dependencies**\n",
+    "\n",
+    "Install dependency `bokeh` using `%pip` cell magic.\n",
+    "\n",
+    "`%pip install` is same as `!/emr/notebook-env/bin/pip install` and are installed in `/home/emr-notebook/`.\n",
+    "\n",
+    "After installation, these libraries are available to any user running an EMR notebook attached to the cluster. Python libraries installed this way are available only to processes running on the master node. The libraries are not installed on core or task nodes and are not available to executors running on those nodes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9204edce",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install matplotlib"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52c15bfc",
+   "metadata": {},
+   "source": [
+    "import dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "52aa4af9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipywidgets import interact\n",
+    "import numpy as np\n",
+    "\n",
+    "from bokeh.io import push_notebook, show, output_notebook\n",
+    "from bokeh.plotting import figure\n",
+    "output_notebook()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d056eab0",
+   "metadata": {},
+   "source": [
+    "Generate x and y axes co-ordinates for sin function.\n",
+    "For x axis, take an array of evenly spaced numbers and calculate corresponding y axis co-ordinate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e5585b19",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = np.linspace(0, 2*np.pi, 2000)\n",
+    "y = np.sin(x)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9654d4b3",
+   "metadata": {},
+   "source": [
+    "Define a graph and draw a line with sin as default function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1035912",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "p = figure(title=\"simple line example\", plot_height=300, plot_width=600, y_range=(-5,5),\n",
+    "           background_fill_color='#efefef')\n",
```

Comment:
Single quotes don't convert to html since they are not escaped. Please review the entire notebook and add only double quotes (").

Reply:
fixed in the new revision.